### DISCUSSION TITLE
Cannot upgrade from 9.9.0 to 9.9.1

### DISCUSSION THREAD

**Discussion by mmmaxxxe** [directus/directus#13077] [Archive]
Hello,
as already done in the past i upgraded directus

sudo npm -u <myuser> update
sudo npx -u <myuser> directus database migrate:latest

and restarted several times. No errors from the console.

at this time the settings pane shows "directus 9.9.0" as current version. 
Any ideas?

**Comment by rijkvanzanten** [comment]
Sounds to me like `npm update` either didn't update Directus correct, or the restart didn't actually restart the Directus process 🤔
